app/models/comment.rb

```diff
@@ -232,52 +232,40 @@ def shorten_urls!
   end
 
   def calculate_score
-    update_column(:score, BlackBox.comment_quality_score(self))
-    update_column(:spaminess_rating, BlackBox.calculate_spaminess(self))
-    root.save unless is_root?
+    Comments::CalculateScoreJob.perform_later(id)
   end
-  handle_asynchronously :calculate_score
 
   def after_create_checks
     create_id_code
     touch_user
   end
 
   def create_id_code
```

Comment:
For all the methods that used to be asynchronous, we need to keep the `%method_name%_without_delay` methods for a while. Please, read the comment about it: https://github.com/thepracticaldev/dev.to/issues/2497#issuecomment-485396688